load/UpdateDBPLanguageTranslation.py

```diff
@@ -85,26 +89,65 @@ def _updateOrInsertLangTranslationsLowerPriority(self, languageNames, languageSo
                     # The column name that belongs to language_translations entity is a varchar with limit of 80 characters
                     languageName = languageName[:80]
 
-                    translationIdWithSameLangAndName = self.getTranslationByLangName(languageName, languageSourceId, languageTranslationId)
+                    row = self.getTranslationByLangName(languageName, languageSourceId, languageTranslationId)
 
                     tableName = "language_translations"
 
-                    if translationIdWithSameLangAndName != None:
+                    if row != None:
+                        (translationIdWithSameLangAndName, oldPriority) = row
                         langExistsByNameAndId = self.getTranslationByLangNameIdAndPriority(languageName, languageSourceId, languageTranslationId, priority)
+                        langEthNameId = self.getTranslationByLangNameIdAndPriority(languageName, languageSourceId, languageTranslationId, self.ETHNOLOGUE_NAME_PRIORITY)
 
-                        if langExistsByNameAndId == None:
+                        if langExistsByNameAndId == None and langEthNameId == None:
                             languageName = languageName.replace("'", "\\'")
-                            attrNames = ("priority",)
                             pkeyNames = ("language_source_id", "language_translation_id", "name", "id")
-                            updateRows.append((priority, languageSourceId, languageTranslationId, languageName, translationIdWithSameLangAndName))
-                            self.dbOut.update(tableName, pkeyNames, attrNames, updateRows)
+                            updateRows.append(("priority", priority, oldPriority, languageSourceId, languageTranslationId, languageName, translationIdWithSameLangAndName))
+                            self.dbOut.updateCol(tableName, pkeyNames, updateRows)
                     else:
                         languageName = languageName.replace("'", "\\'")
                         attrNames = ("name", "priority")
                         pkeyNames = ("language_source_id", "language_translation_id")
                         insertRows.append((languageName, priority, languageSourceId, languageTranslationId))
                         self.dbOut.insert(tableName, pkeyNames, attrNames, insertRows)
 
+    def _updateOrInsertLangTranslationsEthNamePriority(self, languageName, languageSourceId, languageTranslationId, priority):
```

Comment:
rename to _updateOrInsertLangTranslationsEthName

Reply:
@bradflood , it has been changed as you can see in the following line:

https://github.com/faithcomesbyhearing/dbp-etl/pull/95/files#diff-d21d9d4001c386f3021901b139bfa9036284349ef9f6a39ccb598a08c7234787R113